### DISCUSSION TITLE
建议添加一些ui布局工具

### DISCUSSION THREAD

**Discussion by speadk888** [he3-app/he3-feedback#166] [Bug Report]
建议大佬添加ui布局工具，比如python的tk模块可视化布局，可以参考
https://github.com/iamxcd/tkinter-helper

**Comment by lyzhang1999** [comment]
> 建议大佬添加ui布局工具，比如python的tk模块可视化布局，可以参考
> 
> https://github.com/iamxcd/tkinter-helper

请问您是希望有一个可视化的功能来创建小工具吗？
